src/js/bindings/role.js

```diff
@@ -0,0 +1,21 @@
+import { getFilteredCharactersByAlignment } from '../utilities/characterFilters';
+
+export function bindCharacterFilterRole(fn, characters) {
+  document.querySelectorAll('input[name="role"]').forEach((input) => {
+    input.addEventListener('change', (e) => {
+      e.preventDefault();
```

Comment:
I think `e.preventDefault();` is unnecessary here. Usually it's used for situations where the default behavior does something unwanted, like reloading the page when you submit a form. Radio buttons should be fine to change.